modules/base/rendering/renderablesphere.cpp

```diff
@@ -161,6 +177,12 @@ namespace {
         // [[codegen::verbatim(FadeOutThresholdInfo.description)]]
         std::optional<float> fadeOutThreshold [[codegen::inrange(0.0, 1.0)]];
 
+        // [[codegen::verbatim(TransferFunctionPathInfo.description)]]
+        std::optional<std::filesystem::path> colorMap;
```

Comment:
Same here, please make sure that the property and this variable are consistent, at least if using the `verbatim` for documentation. 

I suggest changing the property from "transfer function" to "color map", since this will be understandable for a larger variety of users